processing/src/main/java/io/druid/query/aggregation/JavaScriptAggregatorFactory.java

```diff
@@ -353,6 +354,15 @@ public double aggregate(final double current, final BaseObjectColumnValueSelecto
         return Context.toNumber(res);
       }
 
+      private Object[] arrayToObjectArray(Object array)
+      {
+        final Object[] objectArray = new Object[Array.getLength(array)];
+        for (int j = 0; j < Array.getLength(array); j++) {
```

Comment:
Maybe this line of code can become this way?

```java
int len = Array.getLength(array);
final Object[] objectArray = new Object[len];
for (int j = 0; j < len; j++) {...}
```

Cuz, `for (int j = 0; j < Array.getLength(array); j++) {...}` this code style will call `Array.getLength` method for each `for` loop.

Reply:
Thanks, extracted a variable.